src/main/java/net/sf/jabref/importer/fileformat/BibTeXMLHandler.java

```diff
@@ -32,17 +31,17 @@
  */
 class BibTeXMLHandler extends DefaultHandler {
 
+    private static final String BIBTEXML_URI = "http://bibtexml.sf.net/";
+
     private List<BibEntry> bibitems;
 
     private BibEntry b; // the entry being read
 
     // XML parsing stuff
     private String currentChars;
 
+
     public List<BibEntry> getItems() {
-        if (bibitems == null) {
-            return Collections.emptyList();
-        }
```

Comment:
Then, we add a ridiculous test case for covering these three lines of code? Currently, these lines are never used.